.gitignore

```diff
@@ -38,6 +38,8 @@ captures/
 .idea/tasks.xml
 .idea/gradle.xml
 .idea/assetWizardSettings.xml
+.idea/modules.xml
+.idea/navEditor.xml
```

Comment:
Seen it also in https://gist.github.com/iainconnor/8605514
Might be a good thing to do then